persistence/persistence-api/src/main/java/org/cbioportal/persistence/util/CustomKeyGenerator.java

```diff
@@ -32,26 +32,31 @@
 
 package org.cbioportal.persistence.util;
 
-import java.lang.reflect.Method;
-import java.util.Arrays;
-import java.util.stream.Collectors;
-
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import org.cbioportal.model.util.Select;
 import org.cbioportal.persistence.CacheEnabledConfig;
+import org.cbioportal.persistence.StudyRepository;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.cache.interceptor.KeyGenerator;
 import org.springframework.util.DigestUtils;
 
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
 
 public class CustomKeyGenerator implements KeyGenerator {
     public static final String DELIMITER = "_";
```

Comment:
I just realized that we have two public constants named `DELIMITER`. That's my fault, but let's fix it. Can we rename this? Maybe `CACHE_KEY_PARAM_DELIMITER`?

Reply:
Done